sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -197,19 +197,15 @@ Both `client-id` and `app-id-uri` can be used to verify access token. `app-id-ur
 
 * Step 3: Write Java code:
 
-Write your own adapter class which extends `WebSecurityConfigurerAdapter`, 
-and `AzureJwtBearerTokenAuthenticationConverter` need to be included.
+Write your own adapter class which extends `AADResourceServerSecurityConfigurerAdapter`.
```

Comment:
Describe that if user does not need to customize, aad-starter will provide a default one. Just like web application did.